app/models/apple/publisher.rb

```diff
@@ -57,9 +57,6 @@ def publish!
 
       # success
       SyncLog.create!(feeder_id: public_feed.id, feeder_type: :feeds, external_id: show.apple_id)
-    rescue Apple::ApiError => e
-      SyncLog.create!(feeder_id: public_feed.id, feeder_type: :feeds)
```

Comment:
No more `SyncLog` records for these?

Reply:
Retrying here gets us another log entry -- which is effectively the same?   The sync logs are doing a bit less than I had hoped. But still provide the reference. 

A 1:1 apple state model here might make sense, but the show and episodes need it the least.